go/example_code/s3/delete-buckets-by-prefix/s3_delete_buckets_test.go

```diff
@@ -0,0 +1,146 @@
+/*
+   Copyright 2010-2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+   This file is licensed under the Apache License, Version 2.0 (the "License").
+   You may not use this file except in compliance with the License. A copy of
+   the License is located at
+
+    http://aws.amazon.com/apache2.0/
+
+   This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+   CONDITIONS OF ANY KIND, either express or implied. See the License for the
+   specific language governing permissions and limitations under the License.
+*/
+package main
+
+import (
+    "fmt"
+    "os"
+    "strconv"
+    "strings"
+    "testing"
+    "time"
+
+    guuid "github.com/google/uuid"
+
+    "github.com/aws/aws-sdk-go/aws"
+    "github.com/aws/aws-sdk-go/aws/session"
+    "github.com/aws/aws-sdk-go/service/s3"
+    "github.com/aws/aws-sdk-go/service/s3/s3manager"
+)
+
+func createBucket(sess *session.Session, bucketName string) error {
+    // Create Amazon S3 service client
+    svc := s3.New(sess)
+
+    // Create the S3 Bucket
+    _, err := svc.CreateBucket(&s3.CreateBucketInput{
+        Bucket: aws.String(bucketName),
+    })
+    if err != nil {
+        return err
+    }
+
+    // Wait until bucket is created before finishing
+    err = svc.WaitUntilBucketExists(&s3.HeadBucketInput{
+        Bucket: aws.String(bucketName),
+    })
+    if err != nil {
```

Comment:
might as well return svc.WaitUntilBucketExists(..)

Reply:
Good point, I'll change the code.